classes/core/PKPServicesContainer.inc.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @file classes/core/ServicesContainer.inc.php
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2000-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class ServicesContainer
+ * @ingroup core
+ * @see Core
+ *
+ * @brief Pimple Dependency Injection Container.
+ */
+
+require_once(dirname(__FILE__) . '/../../lib/vendor/autoload.php');
+
+abstract class PKPServicesContainer {
+
+	/** @var Pimple\Container Pimple Dependency Injection Container */
+	private static $instance = null;
+
+	protected $container = null;
+
+	/**
+	 * private constructor
+	 */
+	private function __construct() {
+		require_once(dirname(__FILE__) . '/../../lib/vendor/pimple/pimple/src/Pimple/Container.php');
```

Comment:
@asmecher : pimple is a dependency installed using composer. This is how other composer installed libraries seem to be instantiated. In order to avoid this, I believe we need to required `vendor/autoload.php` in bootstrap stage.